.github/ISSUE_TEMPLATE/--documentation-update.yaml

```diff
@@ -1,7 +1,7 @@
 name: Documentation Update📄
 description: Suggest an improvement/addition in the Keploy Server Docs.
 title: "[docs]: "
-labels: [docs]
```

Comment:
"docs" is fine as shorter/crisp labels are always better. Also, until now the issues were created with the same label so there's no point changing this and distributing the issues even further.

Reply:
Actually, we don't have ``docs`` label in any of repository, so whenever a person opens an issue with this template Docs Label won't be added.

Reply:
In that case, what happens to the existing issues with [docs] prefix? how about renaming documentation label to docs? or we can edit the existing issues prefix.

Reply:
We never had the Docs label so even in existing issues with [docs] prefix, there won't be any changes. Through this prefix won't change only labels would be.

![image](https://user-images.githubusercontent.com/53110238/232176187-760f765e-db71-4775-8f45-1f280cb55209.png)